com.xycz.simple_live.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.xycz.simple_live</id>
+
+  <name>Simple Live</name>
+  <summary>Watch live stream in a simple way</summary>
+
+  <developer_name>xiaoyaocz</developer_name>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+
+  <categories>
+    <category>AudioVideo</category>
+    <category>Audio</category>
+    <category>Video</category>
+  </categories>
+
+  <description>
+    <p>Watch live stream in a simple way.</p>
+    <p>Simple Live is a cross-platform application that allows you to watch live streams from
+      various platforms, including Huya, Douyu, Bilibili, and Douyin. It's available on Android,
+      iOS, Windows, macOS, Linux, and Android TV.</p>
+  </description>
+
+  <launchable type="desktop-id">com.xycz.simple_live.desktop</launchable>
+
+  <url type="bugtracker">https://github.com/xiaoyaocz/dart_simple_live/issues</url>
+  <url type="vcs-browser">https://github.com/xiaoyaocz/dart_simple_live</url>
+
+  <content_rating type="oars-1.1" />
+
+  <screenshots>
+    <screenshot type="default">
+      <image xml:space="preserve">https://raw.githubusercontent.com/xiaoyaocz/dart_simple_live/refs/heads/master/assets/screenshot_light.jpg</image>
```

Comment:
never reference a file from the tip of a branch. it's not immutable.

Reply:
Updated in the latest commit.